app/models/concerns/emojifier.rb

```diff
@@ -0,0 +1,14 @@
+module Emojifier
+  def emojify
+    self.content.gsub(/:([\w+-]+):/) do |match|
+      if emoji = Emoji.find_by_alias($1)
```

Comment:
Assignment in condition - you probably meant to use `==`.<br>Avoid the use of Perl-style backrefs.